app/javascript/src/components/Dashboard/Notes/DropMenu.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+import { MenuVertical } from "neetoicons";
+import { Dropdown } from "neetoui";
+
+const DropMenu = ({ onClick }) => {
```

Comment:
What does `DropMenu` mean? Can I drop files into this menu?

 Or did you mean this is a `DropdownMenu`? If yes, I wonder what advantage you gained by omitting the `down` keyword from `Dropdown` and just keeping it as `Drop`.

Lemme know your thought process. Thanks.

Reply:
The reason I named it `DropMenu` was because `neetoUI` contained the `Dropdown` component. So I couldn't call it `Dropdown`. I thought of naming it `DropdownMenu` but you had posted in Slack that we should avoid verbose component names. I thought `DropMenu` would be an easy guess as no other component resembled it anyway. 

I will rename it to `DropdownMenu`.